src/cephadm/cephadm

```diff
@@ -2079,6 +2079,9 @@ def command_shell():
             '-e', "PS1=%s" % CUSTOM_PS1,
         ]
 
+    for e in args.env:
+        container_args += ['-e', e]
+
```

Comment:
Can you move this down into `CephContainer`?  Like adding a new argument to `CephContainer.__init__()` called envs?